politeiawww/cmd/politeiavoter/politeiavoter.go

```diff
@@ -1436,31 +1349,39 @@ func (c *ctx) verifyVote(vote string) error {
 	// See if vote is ongoing
 	vsr, err := c._summary(vote)
 	if err != nil {
-		return fmt.Errorf("could not obtain proposal status: %v\n",
+		return fmt.Errorf("could not obtain proposal status: %v",
 			err)
 	}
 	vs, ok := vsr.Summaries[vote]
 	if !ok {
 		return fmt.Errorf("proposal does not exist: %v", vote)
 	}
-	if vs.Status != v1.PropVoteStatusFinished {
-		return fmt.Errorf("proposal vote not finished: %v\n", vs.Status)
+	if vs.Status != tkv1.VoteStatusFinished &&
+		vs.Status != tkv1.VoteStatusRejected &&
+		vs.Status != tkv1.VoteStatusApproved {
+		return fmt.Errorf("proposal vote not finished: %v", vs.Status)
```

Comment:
Any kind of status code in the new APIs will have a human readable equivalent.  Use these human readble statuses when logging them.

```
		return fmt.Errorf("proposal vote not finished: %v",
			tkv1.VoteStatuses[vs.Status])
```